mycroft/audio/audioservice.py

```diff
@@ -187,6 +187,8 @@ def load_services_callback(self):
         self.bus.on('mycroft.audio.service.next', self._next)
         self.bus.on('mycroft.audio.service.prev', self._prev)
         self.bus.on('mycroft.audio.service.track_info', self._track_info)
+        self.bus.on('mycroft.audio.service.seek.forward', self._seek_forward)
+        self.bus.on('mycroft.audio.service.seek.backward', self._seek_backward)
```

Comment:
I think `mycroft.audio.service.seek_forward` and `mycroft.audio.service.seek_backward` will fit better with the others.

Reply:
agreed, will make changes